### PR TITLE
Uml 1056 and 1100 : Error pages and survey links corrected

### PR DESCRIPTION
**Pull request by SeemaMenon**
# Purpose

https://opgtransform.atlassian.net/browse/UML-1056
https://opgtransform.atlassian.net/browse/UML-1100

Fixes UML-1056 and UML-1100

## Approach

_Explain how your code addresses the purpose of the change_

## Learning

_Any tips and tricks, blog posts or tools which helped you. Plus anything notable you've discovered about the Use a Lasting Power of Attorney service_

## Checklist

* [X] I have performed a self-review of my own code
* [ ] I have added relevant logging with appropriate levels to my code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [ ] I have added tests to prove my work
* [ ] I have added welsh translation tags and updated translation files
* [ ] The product team have tested these changes

### CODE REVIEW

File: service-front/app/src/Viewer/templates/viewer/viewer-accessibility-statement.html.twig [hunk 1]
```diff
@@ -5,7 +5,7 @@
 {% block content %}
     <div class="govuk-width-container">
 
-        {{ include('@actor/partials/new-use-service.html.twig') }}
+        {{ include('@partials/new-service.html.twig') }}
```
Comment (GemTay) [inline service-front/app/src/Viewer/templates/viewer/viewer-accessibility-statement.html.twig:8]: I think the new-service.html.twi should be renamed to new-view-service.html.twig to match follow the same pattern as the use side, and then moved it from common to the viewer templates directory

File: service-front/app/src/Common/templates/error/error.html.twig [hunk 2]
```diff
@@ -4,22 +4,28 @@
...
+                </p>
+                <p class="govuk-body">
+                    {%trans with {'%link%': path('contact-us')} %}
+                        You can <a class="govuk-link" href="%link%">contact us</a> if you have any questions.
+                    {% endtrans %}
+                </p>
 
+{#                {% if status == 404 %}#}
+{#                    {{ include('@error/404.html.twig') }}#}
+{#                {% endif %}#}
```
Comment (GemTay) [inline service-front/app/src/Common/templates/error/error.html.twig:26]: Comments to be removed

File: service-front/app/src/Common/templates/error/error.html.twig [hunk 3]
```diff
@@ -4,22 +4,28 @@
 
 {% block content %}
 <div class="govuk-width-container">
-    {{ include('@partials/new-service.html.twig') }}
```
Comment (GemTay) [inline service-front/app/src/Common/templates/error/error.html.twig:7]: How come the feedback banner has been removed from the error page?

**Review by GemTay — CHANGES_REQUESTED** [review]
Few changes to be made - also I think you may have forgotten the translations as the error page doesn't appear in welsh with the cy prefix and I can't see any translation files committed either

File: service-front/app/src/Common/templates/error/error.html.twig (hunk 3 above)
Comment (SeemaMenon) [reply service-front/app/src/Common/templates/error/error.html.twig:7]: As per Lewis and Jude suggestion on the ticket.

**Comment by SeemaMenon** [comment]
> Few changes to be made - also I think you may have forgotten the translations as the error page doesn't appear in welsh with the cy prefix and I can't see any translation files committed either

SSM: There is another ticket for the translations. The translations wouldn't work at the moment for the error pages. Also Lewis is only yet to provide translations which can now be added to the other ticket. Sarah is aware I guess.

File: service-front/app/src/Viewer/templates/viewer/viewer-accessibility-statement.html.twig (hunk 1 above)
Comment (SeemaMenon) [reply service-front/app/src/Viewer/templates/viewer/viewer-accessibility-statement.html.twig:8]: Done...and a good point.

File: service-front/app/src/Common/templates/error/error.html.twig (hunk 2 above)
Comment (SeemaMenon) [reply service-front/app/src/Common/templates/error/error.html.twig:26]: My bad

**Review by GemTay — APPROVED** [review]
LGTM 👍

**Pull request merged**